testing/wuanalysis/ecl/anahashdistrib1.ecl

```diff
@@ -19,9 +19,14 @@
 //
 // NOTE: For faster nodes, it may be necessary to increase the size of testfile
 
-IMPORT * From common;
+//nokey
+//noroxie
+//nohthor
 
-visits := DATASET(testfile1, layout_visits, THOR);
+IMPORT $.common.Files as Files;
```

Comment:
picky: possibly simpler to say

import Files from $.common;

but fine as it stands